packages/hardhat-plugin/src/buildIgnitionProvidersFrom.ts

```diff
@@ -49,6 +49,20 @@ export function buildIgnitionProvidersFrom(hre: HardhatRuntimeEnvironment) {
       },
     },
     config: new ConfigWrapper(),
+    accounts: {
+      getAccounts: async () => {
+        try {
+          return (await hre.network.provider.request({
+            method: "eth_accounts",
+          })) as string[];
+        } catch {
+          return [];
```

Comment:
I would be tempted not to catch this. I suspect if we can't get accounts we should stop, whether in tests or a full deploy.

Reply:
I pushed the removal of this catch as it was the last outstanding change.